### PR TITLE
[AST] Implement to_sql func for OrderByExpr

### LINKED ISSUE #809: [AST] Implement to_sql func for OrderByExpr
## Description 

Implement to_sql func for OrderByExpr

## Task

```rust
#[test]
fn to_sql_order_by_expr() {
    let actual = "foo ASC".to_string();
    let expected = OrderByExpr {
        expr: Expr::Identifier("foo".to_string()),
        asc: Some(true),
    }
    .to_sql();
    assert_eq!(actual, expected);

    let actual = "foo DESC".to_string();
    let expected = OrderByExpr {
        expr: Expr::Identifier("foo".to_string()),
        asc: Some(false),
    }
    .to_sql();
    assert_eq!(actual, expected);

    let actual = "foo".to_string();
    let expected = OrderByExpr {
        expr: Expr::Identifier("foo".to_string()),
        asc: None,
    }
    .to_sql();
    assert_eq!(actual, expected);
}
```

### PR DESCRIPTION
**Pull request by CEOJINSUNG**
Resolve #809

### CODE REVIEW

**Review by ever0de — APPROVED** [review]
Looks good :+1:

**Review by panarch — CHANGES_REQUESTED** [review]
Current ToSql impl of `VALUES` is different from what it actually does.
e.g.
```rust
VALUES (1, NULL), (2, 'a'), (3, 4)
```

More information is here in `test-suite/ values.rs`
ref. https://github.com/CEOJINSUNG/gluesql/blob/features/query-to-sql/test-suite/src/values.rs

File: core/src/ast/mod.rs [hunk 1]
```diff
@@ -224,9 +224,10 @@ impl ToSql for Statement {
                 column: _column,
             } => {
                 format!(
-                    "CREATE INDEX {} ON {} (..order_by_expr..)",
+                    "CREATE INDEX {} ON {} {}",
                     name.to_sql(),
-                    table_name.to_sql()
+                    table_name.to_sql(),
+                    _column.to_sql()
```
Comment (panarch) [inline core/src/ast/mod.rs:230]: Now the `_column` is being used, so we can remove `_` underscore prefix.
`column.to_sql()` would be enough

**Review by panarch — APPROVED** [review]
Awesome! Looks all nice, now `CREATE INDEX` ToSQL impl has completed :D Thanks a lot!

**Pull request merged**